white-paper.md

```diff
@@ -88,7 +88,7 @@ tea’s mission is to empower open-source communities and ensure their contribut
 
 ## Package Maintainers
 
-Package maintainers must make sure their software continues to deliver increasing value as the industry evolves.
+Package maintainers must make sure that their software continues to deliver increasing value as the industry evolves.
```

Comment:
“that” is considered a word you can leave out to improve how the sentence reads.